### DISCUSSION TITLE
How to configure the RecordBuilder when using RecordInterface?

### DISCUSSION THREAD

**Discussion by marcusti** [Randgalt/record-builder#63] [Q&A] [answered]
Hey, thanks for creating this project! Looks like a fine replacement for `Immutables`.

I was wondering, is there a way to configure the `RecordBuilder` when using `RecordInterface`? I would like to pass options like `useImmutableCollections = true`.

**Comment by Randgalt** [comment, marked as answer]
OK - it looks like this is an oversight. I'll open an issue to fix this.

**Reply by Randgalt** [reply]
https://github.com/Randgalt/record-builder/issues/64
